velox/functions/lib/MapFromEntries.cpp

```diff
@@ -94,14 +97,14 @@ class MapFromEntriesFunction : public exec::VectorFunction {
     auto& inputValueVector = inputArray->elements();
     exec::LocalDecodedVector decodedRowVector(context);
     decodedRowVector.get()->decode(*inputValueVector);
-    // If the input array(unknown) then all rows should have errors.
     if (inputValueVector->typeKind() == TypeKind::UNKNOWN) {
-      try {
-        VELOX_USER_FAIL(kErrorMessageEntryNotNull);
-      } catch (...) {
-        context.setErrors(rows, std::current_exception());
+      if (throwOnNull_) {
```

Comment:
Can we use `if (inputValueVector->typeKind() == TypeKind::UNKNOWN && throwOnNull_)`?

Reply:
Not quite. When type is `UNKNOWN`, we need to do code after this check to construct the `map(unknown, unknown)` and compose these 2 checks is not necessary.